shopinvader/views/partner_view.xml

```diff
@@ -7,6 +7,14 @@
         <field name="arch" type="xml">
             <notebook position="inside">
                 <page name="shopinvader" string="Shopinvader">
+                    <group name="extras">
```

Comment:
TODO: a button is needed for the address as well. Best thing: quick button on the kanban list of contacts/addresses. Fallback: a button on the form.